os/android/iahwc2.cpp

```diff
@@ -853,42 +896,13 @@ HWC2::Error IAHWC2::HwcDisplay::SetColorModeWithRenderIntent(int32_t mode,
   return HWC2::Error::None;
 }
 
-HWC2::Error IAHWC2::HwcDisplay::GetRenderIntents(int32_t mode,
```

Comment:
this method is not changed, right? if so, please clear the patch to keep it unchanged